data/campaigns/tutorial/_main.cfg

```diff
@@ -6,6 +6,31 @@
 
 # wmlscope: set export=no
 
+[campaign]
+    id=Tutorial
+    name= _ "Battle Training"
+    icon="data/campaigns/tutorial/images/units/quintain.png~RC(magenta>blue)"
+    image="data/campaigns/tutorial/images/units/quintain.png~RC(magenta>blue)~XBRZ(4)"
+    abbrev= _ "Tutorial"
+    rank=1
+    # put it last when sorted by date
+    year="9999 AF"
```

Comment:
year=519 YW
is also an option (right before HttT)

Reply:
The tutorial isn't canonical lore-wise, so I think either first or last would be better.